src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -214,12 +214,12 @@ internal static string GetRuntimeVersion()
 
         internal static Runtime GetCurrentRuntime()
         {
+            if (IsMono)
+                return Runtime.Mono;
             if (IsFullFramework)
                 return Runtime.Clr;
             if (IsNetCore)
                 return Runtime.Core;
-            if (IsMono)
-                return Runtime.Mono;
```

Comment:
Why do we change the order of conditions here? If we need a specific order, it would be nice to have a comment which explains the requirements.

Reply:
@AndreyAkinshin I did it on the base of the order in RuntimeInformation.GetRuntimeVersion(). When I ran my tests on Mono, RuntimeInformation.GetCurrentRuntime() detected runtime as CLR.

Reply:
In this case, we definitely need a comment.